worldedit-forge/src/main/java/com/sk89q/worldedit/forge/ForgePlayer.java

```diff
@@ -203,6 +204,11 @@ public void setFlying(boolean flying) {
         }
     }
 
+    @Override
+    public Locale getLocale() {
+        return Locale.forLanguageTag(player.language.replace('_', '-'));
```

Comment:
Just thinking of this down here, but can we add, maybe to TextUtils or something, this language mapping function? I have a hunch it will need fixing/tuning in the future, and I'd rather not replace the ~4 instances of it :)

When I say this I mean the whole mapping from `player.language` to a `Locale`, so it would include `forLanguageTag` in the function body.